### PR TITLE
feature/BDD-GWT-IP2-181-Verification-of-activation-code

### PR DESCRIPTION
**Pull request by katkoch**
scenarios for verification of activation code

### CODE REVIEW

File: RestSharpProject/Features/VerificationOfActivationCode.feature [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	# Link JS: https://tracker.intive.com/jira/browse/IP2-136
+
+Background:
+Given Endpoint is set
+
+@ignore
+#manual test
+# Link do Testu_1 w Zephyr
+Scenario: REGISTRATION_FORM_1_IP2-292_Email_is_positively_verified
+	Given User is in data base
```
Comment (AdamAndrysek) [inline RestSharpProject/Features/VerificationOfActivationCode.feature:15]: database. It should be corrected also in next scenarios.

File: RestSharpProject/Features/VerificationOfActivationCode.feature [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	And is not activated
```
Comment (AdamAndrysek) [inline RestSharpProject/Features/VerificationOfActivationCode.feature:16]: Maybe better it'd be: 
And his status is INACTIVE

File: RestSharpProject/Features/VerificationOfActivationCode.feature [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	When Client enters the code and User ID
+	And the request is sent to API
+	Then Verification is succesfull, 
+	And return Status is 200
+	And User is activated
```
Comment (AdamAndrysek) [inline RestSharpProject/Features/VerificationOfActivationCode.feature:21]: Maybe: JSON body contains "status": "ACTIVE"
What do you think?

File: RestSharpProject/Features/VerificationOfActivationCode.feature [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+@ignore
+#manual test
+# Link do Testu_1 w Zephyr
+Scenario: REGISTRATION_FORM_1_IP2-292_Email_is_positively_verified
+	Given User is in data base
+	And is not activated
+	When Client enters the code and User ID
+	And the request is sent to API
+	Then Verification is succesfull,
```
Comment (AdamAndrysek) [inline RestSharpProject/Features/VerificationOfActivationCode.feature:19]: Should be "successful", not "succesfull". Please change it also in next scenarios.

**Review by AdamAndrysek — CHANGES_REQUESTED** [review]

**Review by AdamAndrysek — APPROVED** [review]

**Review by kmiecikmarcin — APPROVED** [review]

**Pull request merged**